gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -162,10 +162,15 @@ secrets:
 #kafka:
 #  enabled: false
 
-#  # Host Mode
 #  routingMode: host # default is host. Only host is supported for now.
-#  brokerHostPrefix: broker- # default is broker-
-
+#  # Routing Host Mode
+#  routingHostMode:
+#    brokerPrefix: broker- # default is broker-
+#    # The default domain where the Kafka APIs are exposed. ex: `myapi` will be exposed as `myapi.mycompany.org`
+#    defaultDomain: mycompany.org # Should set according to the public wildcard DNS/Certificate. Default is empty
+#    defaultPort: 9092 # Default public port for Kafka APIs. Default is 9092
+
+#  # Kafka Network settings
 #  port: 9092
```

Comment:
Do you think we could configure `kafka.routingHostMode.defaultPort` with `${kafka.port}` by default ?
@jhaeyaert WDYT?

Reply:
We can't. Despite `port` will often be equals to `kafka.routingHostMode.defaultPort`, it may be different. The `kafka.routingHostMode.defaultPort` represents the port seen by the client (e.g. could be considered similar to Kafka advertising port concept). Internal port could be 9092, while external port would be 9999 for instance.

Reply:
Ok, so maybe we can add some comments on both lines to explain the difference between both "ports"